pkg/agent/nplagent/lib/constants.go

```diff
@@ -0,0 +1,22 @@
+// +build !windows
+
+// Copyright 2020 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package lib
+
+const (
+	NPLEPAnnotation = "npl.k8s.io/endpoints"
```

Comment:
I am not an expert there, but it seems wrong to use `k8s.io` for an Antrea-specific feature. The K8s documentation states:

> The kubernetes.io/ and k8s.io/ prefixes are reserved for Kubernetes core components.

It seems that it should be `npl.antrea.tanzu.vmware.com` to be consistent with the Antrea APIs

Reply:
Agreed. Or for short, we can consider antrea or antrea.io too.